pomdp.py

```diff
@@ -483,3 +499,14 @@ def get_best_action(self, belief):
         highest_expected_reward = res.max()
         best_action = self.action_nums[res.argmax()]
         return (best_action, highest_expected_reward)
+
+
+def is_numeric(set):
```

Comment:
nit: let's avoid `set` as it's a reserved word (it's a type). and i think this function operates on lists, so maybe `l` or `lst` or ... ?